packages/react-i18n/src/i18n.ts

```diff
@@ -499,9 +500,11 @@ export class I18n {
     const timeZone = options?.timeZone;
     const time = this.formatDate(date, {
       ...options,
+      ...undefinedDateDisplayProps,
+      timeZoneName: options?.timeZoneName,
       hour: 'numeric',
       minute: '2-digit',
-    }).toLocaleLowerCase();
```

Comment:
this couldn't do anything other than making the time zone name or era lowercase, which is not in line with [polaris](https://polaris.shopify.com/content/grammar-and-mechanics#section-dates-numbers-and-measurements).

Probably was introduced to make am/pm lower case but they are already lowercase